### PR TITLE
C66 changes

### PR DESCRIPTION
**Pull request by darkdh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,12 +43,14 @@
       },
       "muon": {
         "dir": "src/electron",
+        "branch": "C66",
```
Comment (bridiver) [inline package.json]: we don't want this in master

File: package.json [hunk 2]
```diff
@@ -43,12 +43,14 @@
...
         "repository": {
-          "url": "https://github.com/brave/muon.git"
+          "url": "git@github.com:brave/muon.git"
         }
       },
       "browser_laptop": {
         "dir": "src/browser-laptop",
+        "branch": "0.22.x",
```
Comment (bridiver) [inline package.json]: or this

File: package.json [hunk 3]
```diff
@@ -43,12 +43,14 @@
       },
       "muon": {
         "dir": "src/electron",
+        "branch": "C66",
         "repository": {
-          "url": "https://github.com/brave/muon.git"
+          "url": "git@github.com:brave/muon.git"
```
Comment (bridiver) [inline package.json]: I think we should leave this out too

**Comment by darkdh** [comment]
addressed all the comments

**Pull request merged**